### PR TITLE
Add option to block .eth usernames

### PR DESCRIPTION
**Pull request by Ghosts**
Adds an option to block users with .eth in their user names.

Not necessarily expecting this to get merged as I get that people may want to block NFT related things but not necessarily crypto as a whole - but I made these changes so I could personally start blocking `.eth` usernames & figured I'd throw it out there as an option for others!

### CODE REVIEW

File: src/content/autoBlock.ts
Comment (ottomated) [inline src/content/autoBlock.ts]: This would block "beth" because of the `.`. You want `/\.eth$/`, probably

File: src/content/autoBlock.ts
Comment (ottomated) [inline src/content/autoBlock.ts]: Remove console.log

Comment (ottomated) [reply src/content/autoBlock.ts]: Remove the settings check from here, it belongs in the callback with the rest

File: src/content/autoBlock.ts
Comment (ottomated) [inline src/content/autoBlock.ts]: Rename to `isEthUsername`

**Review by ottomated — CHANGES_REQUESTED** [review]
Probably won't merge this regardless, but here are some bugs

File: src/content/autoBlock.ts
Comment (Ghosts) [reply src/content/autoBlock.ts]: lol oops - should go unblock a few elizabeths, bethanys, and beths 😂

File: src/content/autoBlock.ts
Comment (Ghosts) [reply src/content/autoBlock.ts]: lol oops - should go unblock a few elizabeths, bethanys, and beths 😂

**Pull request closed without merging**
